src/Structs/Soa/v1/ActivateSignatureResponse.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace Coverzen\Components\YousignClient\Structs\Soa\v1;
+
+use Coverzen\Components\YousignClient\Database\Factories\v1\ActivateSignatureResponseFactory;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+/**
+ * Class ActivateSignatureResponse.
+ *
+ * @property string|null $id
+ * @property string|null $type
+ * @property array|null $settings
+ * @property bool $optional
```

Comment:
anche questo deve essere nullable, ma sinceramente lo lascerei cosi mettendo in attribute il default a false